src/server/qgsfcgiserverresponse.h

```diff
@@ -68,7 +70,18 @@ class SERVER_EXPORT QgsFcgiServerResponse: public QgsServerResponse
      * \param method The HTTP method (Get by default)
      */
     QgsFcgiServerResponse( QgsServerRequest::Method method = QgsServerRequest::GetMethod );
-    virtual ~QgsFcgiServerResponse();
+
+    /**
+     *  Dummy copy constructor. Needed because of QgsSocketMonitoringThread ptr
```

Comment:
Is this copy ctor required by SIP bindings ?

Reply:
no by cppcheck ... I do not know how to ignore this "warning" so I added the copy constructor and operator=